.github/workflows/python-publish.yaml

```diff
@@ -64,6 +64,7 @@ jobs:
         uses: docker/build-push-action@v4
         with:
           push: true
-          tags: ghcr.io/${{ github.repository }}:${{ needs.deploy.outputs.version }}
-          context: .
-          file: ./Dockerfile
+          tags: ghcr.io/atlanhq/atlan-python:$(cd .. && cat pyatlan/version.txt)
```

Comment:
Not sure in which directory this runs in! I am assuming it to be inside containers directory.
Would need to run the workflow to check this